sshrd.sh

```diff
@@ -12,6 +12,13 @@ ERR_HANDLER () {
 
 trap ERR_HANDLER EXIT
 
+# Check for pyimg4
```

Comment:
This shouldn't be needed since its already checked in palera1n.sh?

Reply:
just incase